models/reviews.models.js

```diff
@@ -47,16 +47,19 @@ exports.fetchReviews = () => {
 
 exports.fetchReviewComments = (review_id) => {
     const queryStr = `SELECT * FROM comments WHERE review_id = $1`
+    const queryStr2 = `SELECT * FROM reviews WHERE review_id = $1`
 
-    return db.query(queryStr, [review_id]).then((response) => {
-        if(!response.rows.length) {
-            return db.query(`SELECT * FROM reviews WHERE review_id = $1`, [review_id]).then((response) => {
-                if(!response.rows.length) {
-                    return Promise.reject({ status: 404, msg: "not found"})
-                };
-                return [];
-            });
-        }
-        return response.rows;
-    });
+    return Promise.all([
```

Comment:
We are still nesting the promises here. Dont add .thens on to the promises in the array. Try removing the .thens and console logging response in your final .then and see if this helps you figure out how we can use Promise.all to stop us from nesting these promise chains